spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/src/test/java/com/example/TraceSampleApplicationIntegrationTests.java

```diff
@@ -57,22 +62,30 @@
 import org.springframework.http.HttpEntity;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpMethod;
-import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 
 /** Verifies that the logged Traces on the sample application appear in StackDriver. */
-@RunWith(SpringRunner.class)
+// Please use "-Dit.trace=true" to enable the tests
+@EnabledIfSystemProperty(named = "it.trace", matches = "true")
+@ExtendWith(SpringExtension.class)
+@TestPropertySource(properties = { "sampleTopic=traceTopic", "login.pwd=k12" })
```

Comment:
This is a good find for static properties. `"login.pwd=k12"` seems extra.

As an experiment, try setting `sampleTopic` and `sampleSubscription` to something different than the values in the main application's `application.properties`.